pytest-selenium/pages/base.py

```diff
@@ -44,3 +44,15 @@ def offscreen_click(self, element):
         # https://stackoverflow.com/a/39918249
         element.send_keys(Keys.ENTER)
         return element
+
+    def offscreen_click_and_wait_for_new_title_to_load(self, element):
```

Comment:
name binds to implementation, how about

```suggestion
    def click_and_wait_for_load(self, element):
```

Reply:
Changed as mentioned